src/lib/mina/fetch.ts

```diff
@@ -757,9 +796,33 @@ export function createActionsList(
         `No action data was found for the account ${publicKey} with the latest action state ${actionState}`
       );
 
-    actionData = actionData.sort((a1, a2) => {
-      return Number(a1.accountUpdateId) < Number(a2.accountUpdateId) ? -1 : 1;
-    });
+    // DEPRECATED: In case the archive node is running an out-of-date version, best guess is to sort by the account update id
+    // As of 2025-01-07, minascan is running a version of the node which supports `sequenceNumber` and `zkappAccountUpdateIds` fields
+    // We could consider removing this fallback since no other nodes are widely used
```

Comment:
> We could consider removing this fallback since no other nodes are widely used

I would be careful with that and I don't think we should. App developers are currently incentivized to run their own services for that type of stuff, so we will never know what users are exactly using. Imo we should just keep it in there, it doesn't hurt

Reply:
> Imo we should just keep it in there, it doesn't hurt

It's a little sloppy because we will retry on _any_ error.  The archive node API responds with `Error: Unknown Error: {}` when this situation is hit.  So we should also improve the error handling on the API side.

But I'll leave it as-is for now.  The most common case will be that that the first request succeeds and this case is never hit.